### DISCUSSION TITLE
过剧情到选项百分百跳当前获取焦点的窗口不是原神

### DISCUSSION THREAD

**Discussion by Redndye** [babalae/better-genshin-impact#239] [Q&A] [answered]
<img width="254" alt="776b8d11f00ce780ee0b734372de1a20" src="https://github.com/babalae/better-genshin-impact/assets/50910148/44880862-868c-48d6-9ac4-dfe6002e9111">

**Comment by Redndye** [comment]
win10系统，开了管理员权限

**Comment by Redndye** [comment]
最新版本

**Reply by Redndye** [reply]
bitblt不行了，换了截图方式解决了

**Reply by Redndye** [reply]
好吧，切换了一下窗口，也不行了

**Comment by Redndye** [comment]
![image](https://github.com/babalae/better-genshin-impact/assets/50910148/c62f8638-9ef8-427b-b34b-7aab9ce3915d)
新报错

**Comment by huiyadanli** [comment]
看上去是窗体句柄获取的不正确的问题，少数人会遇到这种情况。。0.18版本解决过这个问题，但是针对正常用户会报错

试试0.18版本
https://github.com/babalae/better-genshin-impact/releases/tag/0.18.0

**Reply by Redndye** [reply]
也没用，不知道咋回事，以前用的好好的，退版本了也不行了

**Reply by huiyadanli** [reply]
关闭杀毒软件或者其他相关的软件试试？

**Reply by Redndye** [reply]
重装了，问题转到了提示“PaddleOcr预热失败”应该如何解决？，然后解决了

**Reply by Redndye** [reply, marked as answer]
重装了系统😭
